### DISCUSSION TITLE
can i use xml:id ?

### DISCUSSION THREAD

**Discussion by RenatoCaenaro** [buildingSMART/IDS#373] [Q&A]
As an authoring IDS software,
Is it possible to add xml:id attribute to elements? 
https://www.w3.org/TR/xml-id/
I have tried to add some xml:ids in the sample files and they are still valid for the validator.

**Comment by atomczak** [comment]
What do you want to use it for? Can you provide a sample file? The IDS allows the identification of specifications with an 'identifier'  attribute; have you considered that? https://github.com/buildingSMART/IDS/blob/8d4c774c39b298b22c667fce52c08d31957caf0f/Schema/ids.xsd#L240
